code/modules/power/engines/singularity/collector.dm

```diff
@@ -77,9 +77,14 @@ GLOBAL_LIST_EMPTY(rad_collectors)
 			eject()
 			return TRUE
 	else if(iswrench(I))
+		var/turf/T = get_turf(src)
```

Comment:
You can move this thing under the `loaded_tank` check, as we don't need to run the `get_turf` until we're absolutely sure we can actually wrench it down